zoomus/util.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, base_uri=None, timeout=15, **kwargs):
         self.timeout = timeout
         for k, v in kwargs.items():
             setattr(self, k, v)
+        self.session = requests
```

Comment:
This is a little bit alarming and confusing to see without proper context. I think this probably warrants an explanatory comment, or maybe `self.session` could become a property instead? Along the lines of...

```python
@property
def session(self):
    """Provide a session if one is in use; otherwise, provide the requests module directly."""
    if self._session is not None:
        return self._session
    return requests

@session.setter
def session(self, val):
    self._session = val

@session.deleter
def session(self):
    self._session = None
```

Reply:
REALLY fair point. I'll include your suggestions now. This approach seems a lot more sane